mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'mtp_common/user_admin/sign-up-success.html' %}
+{% load i18n %}
+{% load mtp_common %}
+
+{% block whats_next_message %}
+  <p>
+    {% blocktrans trimmed with approver=approver|wrapwithtag:'strong' %}
```

Comment:
you can drop the `with approver=approver|wrapwithtag:'strong'` now as the param isn't used anymore

Reply:
I thought I got rid of that but well spotted. Sorted: 168abff82171c5d2fa570a00ec5f0bc48115b4f6